src/wsiToDcm.h

```diff
@@ -22,51 +22,40 @@
 
 namespace wsiToDicomConverter {
 
+// Structure for wsi2dcm settings
+struct WsiRequest {
+  std::string inputFile = "";  // wsi input
+  std::string outputFileMask = "./";  // path to save generated files
+  int64_t frameSizeX = 500;
+  int64_t frameSizeY = 500;
+  DCM_Compression compression = JPEG;  // compression - jpeg, jpeg2000, raw
+  int32_t quality = 80;
+  int32_t startOnLevel = 0;   //
+  int32_t stopOnLevel = -1;   // limitation which levels to generate
+  std::string imageName = "image";  //
+  std::string studyId;  //
+  std::string seriesId;  // required DICOM metadata
+  std::string jsonFile = "";  // json file with additional DICOM metadata
+  int32_t retileLevels = 0;  // number of levels, levels == 0  means number of
+                            // levels will be readed from wsi file
+  double* downsamples;  // for each level, downsample is size
+                       // factor for each level
+  bool tiled;  // frame organization type
```

Comment:
Can you provide a link that describes the differences between these two types?

Reply:
done